addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1134,9 +1134,12 @@ function listGroupToTable(editable) {
 function normalizeColors($editable) {
     const editable = $editable.get(0);
     for (const node of editable.querySelectorAll('[style*="rgb"]')) {
-        const rgbMatch = node.getAttribute('style').match(/rgb?\(([\d\.]*,?\s?){3,4}\)/g);
+        const rgbMatch = node.getAttribute('style').match(/rgb[a]*?\(([\d\.]*,?\s?){3,4}\)/g);
+        const bg = getComputedStyle(node).backgroundColor;
+        const bgRgbMatch = bg.match(/\((.*)\)/);
+        const bgValues = bgRgbMatch && bgRgbMatch[1].split(',').map(val => parseInt(val.trim() || '0'));
```

Comment:
The variable names here are slightly confusing because here it's the very first place where we really get the actual RGBA values, and that variable does not have rgba in its name... ^^' Shouldn't it be like `bgRgbaValues` ?

Reply:
Yes again lol